lineup.js

```diff
@@ -1,9 +1,14 @@
 function calculateTotalSalary(lineup) {
-  return 0
+  return lineup.reduce((playerSalary, player) => {
+    return playerSalary + player.salary
+  }, 0)
 }
 
 function getPositionCounts(lineup) {
-  return {}
+  return lineup.reduce((playerPosition, player) => {
```

Comment:
Small naming thing. The accumulator is building a new object with all of the positions from the lineup in it and the counts for how many times they appear. The name `playerPosition` implies the single position of a single player. Naming this something like `positions` or `positionCounts` would be a better choice.